v3/src/components/map/models/map-point-layer-model.ts

```diff
@@ -15,6 +15,7 @@ export const MapPointLayerModel = MapLayerModel
     gridModel: types.optional(MapGridModel, () => MapGridModel.create()),
     pointsAreVisible: true, // This is different than layer visibility
     connectingLinesAreVisible: false,
+    pointType: types.optional(types.string, "points"), // "points" or "heatmap"
```

Comment:
I'd be inclined to call this `displayType` and to define it as an enumerated property:
```suggestion
    displayType: types.optional(types.enumeration([...MapPointDisplayTypes]), "points")
```
and then add:
```typescript
export const MapPointDisplayTypes = ['points', 'heatmap'] as const
export type MapPointDisplayType = typeof MapPlaces[number]
```
```